argoverse/evaluation/detection_utils.py

```diff
@@ -0,0 +1,175 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection utilities for the Argoverse detection leaderboard.
+
+Accepts detections (in Argoverse ground truth format) and ground truth labels
+for computing evaluation metrics for 3d object detection. We have five different,
+metrics: mAP, ATE, ASE, AOE, and DCS. A true positive for mAP is defined as the
+highest confidence prediction within a specified euclidean distance threshold
```

Comment:
nitpick, but I think it should be "Euclidean distance" instead of "euclidean distance"